Various/muorsic_Smart tempo and time signature changes.lua

```diff
@@ -1,9 +1,7 @@
 -- @description Smart tempo and time signature changes
 -- @author muorsic
--- @version 1.8
--- @changelog
---   fixed new point not found 
---   fixed empty points with no bpm
+-- @version 1.81
```

Comment:
8 to 81 is a huge increase and would mask any future 1.9 (9 < 81). ReaPack version numbers are not decimal! https://forum.cockos.com/showpost.php?p=1707548

Did you mean 1.8.1?